vipermonkey/core/statements.py

```diff
@@ -4603,6 +4621,30 @@ def eval(self, context, params=None):
                 # Try to handle that.
                 if (log.getEffectiveLevel() == logging.DEBUG):
                     log.debug("Did not find procedure.")
+                
+                # possible script locations for vbs
+                script_locs = ["range.text", "me.content", "activedocument.rage","me.range","me.content.text","me.range.text"]
+
+                if (("Get" in func_name) and ("#" in call_params[0])):
+                    handled_read = False
+                    for key, module in context.globals.items():
+                        if handled_read: break
```

Comment:
It's more readable to place `break` and `continue` statements on their own line, even if that the only line in the if block. The indentation makes it easier to track where control flow is happening.